### PR TITLE
Update Astra free tier storage

### PR DESCRIPTION
**Pull request by norlanp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -950,7 +950,7 @@ Table of Contents
 ## DBaaS
 
    * [airtable.com](https://airtable.com/) — Looks like a spreadsheet, but it's a relational database, unlimited bases, 1,200 rows/base and 1,000 API requests/month
-   * [Astra](https://astra.datastax.com/register) — Cloud Native Cassandra as a Service with 5GB free tier
+   * [Astra](https://astra.datastax.com/register) — Cloud Native Cassandra as a Service with [40GB free tier](https://docs.datastax.com/en/astra/docs/datastax-astra-database-limits.html)
```
Comment (ripienaar) [inline README.md:953]: Please set the link to https://www.datastax.com/products/datastax-astra/  which demonstrates the features etc - better landing place for people from this list.

Then I think https://www.datastax.com/products/datastax-astra/pricing is probably better if you want to link to the plans

Comment (norlanp) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -950,7 +950,7 @@ Table of Contents
 ## DBaaS
 
    * [airtable.com](https://airtable.com/) — Looks like a spreadsheet, but it's a relational database, unlimited bases, 1,200 rows/base and 1,000 API requests/month
-   * [Astra](https://astra.datastax.com/register) — Cloud Native Cassandra as a Service with 5GB free tier
+   * [Astra](https://astra.datastax.com/register) — Cloud Native Cassandra as a Service with [40GB free tier](https://www.datastax.com/products/datastax-astra/pricing)
```
Comment (ripienaar) [inline README.md:953]: If you dont mind please change `https://astra.datastax.com/register` to `https://www.datastax.com/products/datastax-astra/`, the register url does not say much of anything thanks.

**Comment by ripienaar** [comment]
thanks

**Pull request merged**